2023-09-04.mdx

```diff
@@ -0,0 +1,39 @@
+---
+date: 09-04-2023
+---
+
+### What's new?
+
+Discover the latest enhancements and feature updates to improve your experience with Xata.io
+
+- **A new look for Xata.io** - We've given [Xata.io](https://xata.io) a fresh new look, making it even more user-friendly. This redesign includes severeal improvements, such as clearer messaging and a streamlined workflow, all aimed at providing an improved experience for our users.
+- **File Attachments are now available** - We have embedded [file attachments](https://xata.io/file-attachments) directly into our serverless database and improved the experience for building data apps that require binary storage.
+- **New UI experience for CSV import and export** - Introducing an entirely new [CSV import and export](https://xata.io/docs/csv-data/import-data) interface that simplifies your Xata.io experience, enabling you to automatically create new tables by directly importing your data into Xata's database. The export feature allows you to download your data as CSV files.
+- **Support for JSON ingestion** - Basic support for [JSON column](https://xata.io/blog/json-column-type) types is now available, expanding the range of data formats that Xata.io can handle. 
```

Comment:
Similar to JSON, we could probably add this as something we recently shipped. Can't hurt to call out twice.

https://xata.io/docs/concepts/data-model#links-and-relationships